lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -379,3 +382,14 @@
     puts 'No gettext-setup gem found, skipping GettextSetup config initialization' if Rake.verbose == true
   end
 end
+
+namespace :module do
+  desc 'Generate a JSON description of the module contents'
+  task :inspect do
+    raise 'This task does not work with Puppet <= 3' if puppet_3_or_older?
```

Comment:
nit: Not a fan of `_or_older` but can't think of a better name :-(

Perhaps `puppet_3_or_less` or  `unless supports_puppet4_api?`

Reply:
I prefer `_or_older` rather than `_or_less` as it matches the wordage normally used when describing releases - "an older release" vs "a lesser release"

Reply:
.. or perhaps be explicit. `support_inspection?`